docs/tutorials/bson-v4.txt

```diff
@@ -222,8 +222,8 @@ Supported Classes
 
 Core Ruby classes that have representations in the BSON specification and
 will have a ``to_bson`` method defined for them are: ``Object``, ``Array``,
-``FalseClass``, ``Float``, ``Hash``, ``Integer``, ``NilClass``, ``Regexp``,
-``String``, ``Symbol`` (deprecated), ``Time``, ``TrueClass``.
+``FalseClass``, ``Float``, ``Hash``, ``Integer``, ``BigDecimal``, ``NilClass``,
+``Regexp``, ``String``, ``Symbol`` (deprecated), ``Time``, ``TrueClass``.
```

Comment:
It seems like there might not be such a big addition to the docs in terms of supporting **existing** Ruby classes. Is there more that needs to be done here?